docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,140 @@
+
+2. Select the appropriate bug report file.
+
+3. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in the file. After the analysis is done, a Battery Historian stats chart appears:
+
+**Figure: Battery Historian Chart**
+
+![Historian chart](./media/graph1.png)
+
+The Battery Historian chart graph illustrates the battery consumption events over time, where usage of the various device components correlates to battery consumption.
+
+#### Interpret the Graph
+
+ - The vertical axis represents the battery percentage, while the horizontal axis represents time.
+ - The sloping black bar indicates your battery percentage level throughout the day.
+ - The colored line segments represent the various apps, services, and other elements that have used battery life.
+   - For instance, a green bar denotes an active time for WiFi.
+ - You can add more metrics, by clicking on **Add Metrics** and the corresponding metric will appear on the chart. Some of the available metrics are  Charging status, Voltage, Device active, Plug and so on.
+ - The chart generated is intuitive, hovering over any metric name in the first column shows you more information about that metric.
+
+
+The **System Stats** tab includes overall system statistics, such as screen on time, WiFi idle time, and so on. It gives an overview of what is happening with your Tizen device. For example, you can analyze the impact of a particular application on the battery during its use.
+
+**Figure: System Stats**
+
+![System Stats](./media/graph2.png)
+
+### Compare Bug Reports 
+
+The Battery Historian also allows comparative analysis of two bug reports. The analysis can be viewed in the graphs that highlights the key differences in the battery-related metrics. 
+
+It also helps you to compare and contrast the metrics for battery consumption, and explains differences or similarities for any two key battery metrics of the Tizen device. 
+
+The compare feature of the Battery Historian tool helps you understand which metric is responsible for battery drainage, and diagnose the problem more efficiently. 
+
+To compare the bug report files, follow these steps:
+
+1. In Battery Historian Main Window, click **Switch to Bug Report Comparison**.
+
+    The bug report compare window appears: 
+
+    ![Comparewindow](./media/compare1.png)
+
+    The bug report compare window comprises of the **Browse** components that allows you to browse and select bug report files. 
+
+2. Click **Browse.**
+   
+   Select the respective bug report files. 
+   
+   The **Submit** button appears. 
+  
+   ![Browse](./media/submit.png)
+
+4. Click **Submit.**
+
+   Battery Historian takes a few seconds to analyze the data in both the files, and a comparison chart appears: 
+
+   ![Uploadfiles](./media/compare2.png)
+
+   The chart illustrates the metrics from both the bug reports as the colored bars.
+
+## Related Information
+
+- Dependencies  
+  - Tizen Studio 3.8 and Higher
```

Comment:
Question to this line related with https://github.com/Samsung/tizen-docs/pull/1140

It seems Tizen Studio 3.8 is skipped and Tizen 4.0 will be published instead.
Don't we need to update this line to "Tizen Studio 4.0 and Higher" also?

Reply:
yes it should be made to 4.0 instead of 3.8

Reply:
Thanks for confirming.

@nawab-samsung As the supported Tizen Studio version should be changed to 4.0, would you please update the file

```suggestion
  - Tizen Studio 4.0 and Higher
```

Reply:
Reminder. @nawab-samsung Please update the Tizen Studio version as the supported version is changed to 4.0.

Reply:
@mijin85cho thanks, I changed it to 4.0 from 3.8.